### PR TITLE
(DOCSP-19710) FAQ

### PR DESCRIPTION
**Pull request by biniona-mongodb**
## Pull Request Info

FAQ page.

### Issue JIRA link:

https://jira.mongodb.org/browse/DOCSP-19710

### Snooty build log:

https://workerpool-boxgs.mongodbstitch.com/pages/job.html?jobId=61cdf3e3c52f2a460f578686

### Docs staging link (requires sign-in on MongoDB Corp SSO):

https://docs-mongodbcom-staging.corp.mongodb.com/visual-studio-extension/docsworker-xlarge/DOCSP-19710-FAQ/faq/

### Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [x] Did you run a spell-check?
- [x] Did you run a grammar-check?
- [x] Does it render on staging correctly?
- [ ] Are all the links working? **The JIRA link does not work, as I haven't learned where users should submit issues yet. This will be specified as either a cleanup task or Max can provide it during his review of this PR**.
- [x] Are the staging and workerpool job links in the PR description updated?

### If your page documents a concept, does it meet the following criteria?

- [ ] Target the [Jasmin persona](https://drive.google.com/file/d/14FbBOLCVxwSP6M9BK4Nz1Ir9tzxT8_02/view)
- [ ] Target the [Lucas persona](https://drive.google.com/file/d/1J2vqJxo7ldv7OP_obA9Q-avf0o_ju4Lk/view)

## Notes

[This is the Google Document](https://docs.google.com/document/d/1U6TYL5NR_KlwqyM6fjydiQKd4If7_yHfaR1DFAVEcys/edit) where Boris and I are working out the how to talk about the data collected by the MongoDB Analyzer when telemetry is enabled.

### CODE REVIEW

**Review by terakilobyte — APPROVED** [review]
lgtm

**Comment by biniona-mongodb** [comment]
@BorisDog and @mmarcon, this PR is ready for your review. 

@BorisDog, in regards to the telemetry fields, I recorded reasons for divergences from your edits as comments in the [google document](This is the Google Document). Let me know if you disagree with any of my responses to your feedback 🙂 .

File: source/includes/telemetry-data.rst [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+   This file describes all telemetry data collected
+   by the {+product+}. Field names are not included
+   as they are not exposed through the public API.
+
+Your Code
+~~~~~~~~~
+
+The following is the information the {+product+} collects about your code:
+
+- The number of unique data types you use as generic type parameters in your
```
Comment (BorisDog) [inline source/includes/telemetry-data.rst:12]: I did not mention this in the doc: nested types and enums are counted as well. Maybe just "The number of unique data types used in your LINQ and builder expressions" would be more accurate?

**Review by BorisDog — CHANGES_REQUESTED** [review]

File: source/includes/telemetry-data.rst (hunk 1 above)
Comment (biniona-mongodb) [reply source/includes/telemetry-data.rst:12]: Discussed this with Boris on Slack, and we arrived at the phrasing "The number of user defined data types in your provided [mapping](https://mongodb.github.io/mongo-csharp-driver/2.14/reference/bson/mapping/)"

**Review by BorisDog — APPROVED** [review]
LGTM

**Review by mmarcon — APPROVED** [review]

**Pull request merged**